### ISSUE TITLE
Is is still working ? 

### ISSUE THREAD

**Issue by Tyrannas** [ricklupton/rmc#14]
Used it like `rmc e8376dec-0dcd-4065-bc72-28958ac3a976.rm -o output.pdf` and got `ValueError: Wrong header: b'reMarkable .lines file, version=5`. 

Did the last versions break ? I'm actually running on 3.14.1.9

**Comment by punchagan** [comment]
The [parser](https://github.com/ricklupton/rmscene/blob/main/src/rmscene/tagged_block_common.py#L63) seems to only support `version=6`.

**Comment by Azeirah** [comment]
Correct. rmc only supports version 6. Older versions are not supported.

You can try remarks, although the output is not amazing.

**Comment by Seb-sti1** [comment]
Regarding older versions, it seems that going to each page of each file and drawing anything (even if removing afterward) makes the remarkable convert the file to the last version (currently `version=6` - _I have version 3.15.4.2_).

At least it is how I was able to make all my files compatible with this project.
